pkg/azurefile/nodeserver.go

```diff
@@ -147,6 +177,9 @@ func (d *Driver) NodeUnpublishVolume(_ context.Context, req *csi.NodeUnpublishVo
 	if err := CleanupMountPoint(d.mounter, targetPath, true /*extensiveMountPointCheck*/); err != nil {
 		return nil, status.Errorf(codes.Internal, "failed to unmount target %s: %v", targetPath, err)
 	}
+	if err := directvolume.Remove(targetPath); err != nil {
```

Comment:
what does this line do ? can you add a comment? what if there is no kata-cc mount in the targetPath

Reply:
Remove deletes the direct volume path including all the files inside it.
If there is no kata-cc mountinfo present on this path, it will return nil.